e2e/wdio/platform/tests/object-list-item.e2e-spec.ts

```diff
@@ -124,7 +124,7 @@ describe('Object list item suite:', function() {
     });
 
     describe('Check visual regression', function() {
-        it('should check examples visual regression', () => {
+        xit('should check examples visual regression', () => {
```

Comment:
Why xit tests? If test is not passing after introduced change, simply skip failing tests is against currently approved progress. If you need any support with that, please ping any QA team member instead of skip.

Reply:
@AntonOlkhovskyi ping

Reply:
@droshev I pushed new VR images for this component. CI build is now passing.

Reply:
@AntonOlkhovskyi what do you want me to change since you requested changes

Reply:
No, sorry. It is my bad. I thought I did it along with adding the images.